nautobot/docs/index.md

```diff
@@ -57,6 +57,10 @@ Nautobot is built on the [Django](https://djangoproject.com/) Python Web framewo
 | Task queuing       | Redis/Celery/django-rq          |
 | Live device access | NAPALM                          |
 
+The low level diagram displays how data travels through Nautobot's application stack.
+
+![Application stack diagram](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/nautobot_application_stack_low_level.png "Application stack diagram")
```

Comment:
In the low-level diagram, "Database" should perhaps be broken out into "PostgreSQL" and "MySQL"?